packages/dht/src/connection/WebSocket/WebSocketConnector.ts

```diff
@@ -102,18 +101,22 @@ export class WebSocketConnector implements IWebSocketConnectorService {
                     this.attachHandshaker(connection)
                 }
             })
-            await this.webSocketServer.start(this.webSocketPort!, this.webSocketHost)
+            const port = await this.webSocketServer.start(this.webSocketPortRange!, this.webSocketHost)
+            this.selectedPort = port
+            this.connectivityChecker = new ConnectivityChecker(this.selectedPort!)
```

Comment:
What is the difference between `this.connectivityChecker `and the `checkConnectivity` method? Do we need both?

Reply:
checkConnectivity uses the ConnectivityChecker-class.